src/test/resources/org/broadinstitute/hellbender/tools/picard/fasta/testFASTA_WS_4.fasta

```diff
@@ -0,0 +1,284 @@
+>Chromosome
```

Comment:
is this the actual input used? if yes, it should go to a directory picard/fasta/fastq/NormalizeFasta/

if not, remove this file